openpype/hosts/nuke/plugins/publish/validate_rendered_frames.py

```diff
@@ -116,15 +116,17 @@ def process(self, instance):
                 "len(collection.indexes): {}".format(collected_frames_len)
             )
 
-            if ("slate" in instance.data["families"]) \
-                    and (frame_length != collected_frames_len):
-                collected_frames_len -= 1
-                f_start_h += 1
+            if (
+                    "slate" in instance.data["families"]
+                    and frame_length != collected_frames_len
+            ):
+                f_start_h -= 1
+                frame_length += 1
```

Comment:
I know it's pre-existing code, but I'd add a comment as to what these if branches are trying to solve. It seem odd to say the least.

Reply:
variables were improved in #5500